common/on_action/wc_on_actions.txt

```diff
@@ -0,0 +1,105 @@
+﻿on_death = {
+	on_actions = {
+		wc_death_on_actions
+	}
+}
+
+wc_death_on_actions = {
+	effect = {
+		if = {
+			limit = { exists = scope:killer }
+			hurt_scourge_jailor_consequences_effect = { HURT_JAILOR = root SCOURGE = scope:killer }
+			if = {
+				limit = {
+					scope:killer = { is_arthas_trigger = yes }
```

Comment:
Rather then checking that it is arthas, maybe check if it's just one of his children?

Reply:
It may not be

Reply:
Yes, but the funeral event should happen if another of Terenas kids kill him for whatever reason, not just Arthas

Reply:
better like this ?

Reply:
You're referencing `scope:terenas` before it is set. Use `root` instead.

Reply:
And put `scope:killer = {}` around it

Reply:
Now it looks good